packages/harmony/src/storybook/components/RelatedFoundations.tsx

```diff
@@ -0,0 +1,155 @@
+      <Flex gap='l'>
+        <Paper h={48} w={48} borderRadius='s' shadow='near' />
+        <Paper h={48} w={48} borderRadius='s' shadow='mid' />
+        <Paper h={48} w={48} borderRadius='s' shadow='far' />
+      </Flex>
+    ),
+    href: '/?path=/docs/foundations-shadow--documentation'
+  },
+  CornerRadius: {
+    title: 'Corner Radius',
+    description:
+      'Used to give sharp edges a more subtle, rounded effect and style.',
+    href: '/?path=/docs/foundations-corner-radius--documentation',
+    component: (
+      <Flex gap='l'>
+        <Box
+          h={48}
+          w={48}
+          css={(theme) => ({
+            border: '1px solid rgba(0, 0, 0, 0.20)',
+            background: theme.color.background.white,
+            borderRadius: theme.cornerRadius.xs
+          })}
+        />
+        <Box
+          h={48}
+          w={48}
+          css={(theme) => ({
+            border: '1px solid rgba(0, 0, 0, 0.20)',
+            background: theme.color.background.white,
+            borderRadius: theme.cornerRadius.m
+          })}
+        />
+        <Box
+          h={48}
+          w={48}
+          css={(theme) => ({
+            border: '1px solid rgba(0, 0, 0, 0.20)',
+            background: theme.color.background.white,
+            borderRadius: theme.cornerRadius.xl
+          })}
+        />
+      </Flex>
+    )
+  },
+  Spacing: {
+    title: 'Spacing',
+    description:
+      'Ensure layouts are consistent with our intuitive spacing system.',
+    href: '/?path=/docs/foundations-spacing--documentation',
+    component: (
+      <Flex
+        h={48}
+        w={107}
+        alignItems='center'
+        justifyContent='center'
+        css={{ background: 'rgba(255, 0, 0, 0.10)' }}
+      >
+        <Text variant='body' size='xs' css={{ color: '#F00' }}>
+          32
+        </Text>
+      </Flex>
+    )
+  },
+  Icons: {
+    title: 'Icons',
+    description:
+      'Visually communicate intent. Icons that are crisp, clear, and coherent.',
+    href: '/?path=/docs/foundations-icons--documentation',
+    component: <IconNote height={48} width={48} color='default' />
+  },
+  Motion: {
+    title: 'Motion',
+    description:
+      'Animations that brings the UI to life and ensuring usability.',
+    href: '/?path=/docs/foundations-motion--documentation',
+    component: <Button color='blue'>Quick!</Button>
+  }
+}
+
+type RelatedFoundationsProps = {
+  foundationNames: Array<keyof typeof relatedFoundationsMap>
+}
+
+export const RelatedFoundations = (props: RelatedFoundationsProps) => {
+  const { foundationNames } = props
+
+  return (
+    <Flex
+      mv='3xl'
+      gap='2xl'
+      css={{ section: { flex: '0 0 calc(33.33% - 32px)' } }}
+      wrap='wrap'
+      justifyContent='space-between'
+    >
+      {foundationNames.map((foundationName) => {
+        const { component, link, description } =
+          relatedFoundationsMap[foundationName]
+        const href = `../?path=/docs/foundations-${link}`
```

Comment:
it looks like the args don't all match up here with the map above. You're using link in some and href in others, and the href ones already have the prefix.

Reply:
nice find, yup will fix